src/main/resources/schema.graphqls

```diff
@@ -1571,6 +1571,22 @@ input ReagentTransferRequest {
     plateType: String!
 }
 
+"""A request to record transfer, dual index and amplification ops."""
+input LibraryPrepRequest {
+    """The work number to associate with these operations."""
+    workNumber: String!
+    """The source labware and new labware states for this request."""
+    sources: [SlotCopySource!]!
+    """The one destination labware for this request, and the description of what is transferred into it."""
+    destination: SlotCopyDestination!
+    """The transfers from aliquot slots to destination slots."""
+    reagentTransfers: [ReagentTransfer!]!
+    """The type of reagent plate involved."""
+    reagentPlateType: String
+    """The measurement to record on slots in the destination."""
+    slotMeasurements: [SlotMeasurementRequest!]!
+}
```

Comment:
Just to review my understanding - 
- There should be two source labware (out of which one is a dual index plate) and one destination labware. 
-  It should allow mapping of slots between any of these two source labware and destination labware. 
- It should allow recording of CQ values for the scanned in source dual index plate

I can see a reagentTransfers array which keep the mapping between dualindex source plate and destination. But, where are we going to keep the mapping of slots between other source and destination labware. Is the same array is used for that as well?

Reply:
SlotCopyDestination includes an array of contents, describing what is transferred from the source labware. The destination is created just like for a SlotCopyRequest. The reagent plate is used just like in a ReagentTransferRequest. (I don't know if Stan-client would treat the reagent barcode as a "source" or as some other special kind of input—whatever it does for ReagentTransferRequest.) The LibraryTransferRequest is a combination of parts of a SlotCopyRequest, ReagentTransferRequest and OpWithSlotCommentsRequest. The cq measurement is recorded on the destination labware.